src/pypromice/process/resample.py

```diff
@@ -53,8 +53,9 @@ def resample_dataset(ds_h, t):
                 
                 df_d[var] = (p_vap.to_series().resample(t).mean() \
                            / es_wtr.to_series().resample(t).mean())*100
-                df_d[var+'_cor'] = (p_vap.to_series().resample(t).mean() \
-                           / es_cor.to_series().resample(t).mean())*100
+                if var+'_cor' in df_d.keys():
```

Comment:
Why can we assume the corrected variable is already present in df_d?

Reply:
Consider using a variable for the corrected variable name.
```suggestion
                corrected_variable_name = f"{var}_cor"
                if corrected_variable_name in df_d.keys():
```

Reply:
> Why can we assume the corrected variable is already present in df_d?

Because the same function can be called on L3 data which has `rh_cor` and L0/L1/L2 data which does not.

>  corrected_variable_name = f"{var}_cor"
There's some discussion ongoing about not calling `rh_cor` for "corrected". We will need to rename at a later point. So no need to focus on this part atm.